packages/completer/style/base.css

```
@@ -137,47 +137,47 @@
 }
 
 .jp-Completer-type[data-color-index='0'] {
-  background: transparent;
+  background: var(--jp-completer-type-background0);
 }
 
 .jp-Completer-type[data-color-index='1'] {
-  background: #1f77b4;
+  background: var(--jp-completer-type-background1);
```

What happens in third party themes which may not have the variables yet? I think this may be breaking so I would add fallback:

```suggestion
  background: var(--jp-completer-type-background1, #1f77b4);
```

---

Done, @krassowski!